ci_framework/roles/set_openstack_containers/templates/update_env_vars.sh.j2

```diff
@@ -0,0 +1,35 @@
+---
```

Comment:
if it's a bash script, you probably don't want to add this line.

Something like:
```Bash
#!/bin/sh
set -eo pipefail
```
might be better.